src/PKSim.Infrastructure/Services/MoBiExportTask.cs

```diff
@@ -91,13 +91,14 @@ private void startMoBiWithFile(Func<string> contentFileFullPathFunc, string opti
 
       private string retrieveMoBiExecutablePath()
       {
+         //Path was defined in application settings? it takes precedence over the configuration
+         if (FileHelper.FileExists(_applicationSettings.MoBiPath))
```

Comment:
@rwmcintosh I am not quite sure how to test this really unless I make this function protected ? thoughts? Or are we happy with status quo (e.g. no tests)

Reply:
FileHelper.FileExists is an action that can be replaced in Context and then cleaned up

`When_exporting_a_simulation_to_MoBi_and_mobi_is_not_found_on_the_system`